metadata-ingestion/examples/affirm_artifact/3rd-party.recipe.yml

```diff
@@ -0,0 +1,12 @@
+source:
+    type: affirm-artifact
+    config:
+        directory: '/Users/tao.sun/tao/datahub-metadata/artifacts/3rd_party'
+        platform: 'affirm3rdParty'
+        platform_instance: ''
```

Comment:
Is this a required parameter? Can it be `None` for now? We decided not to leverage `platform_instance`.

Reply:
yeah, it's optional and we can remove them here.